Running examples - cannot find type FillVertex in crate tesselation

```bash
~/personal_projects/rust/vendor/felin (master)$ cargo run --example text
   Compiling felin v0.1.0 (/home/roman/personal_projects/rust/vendor/felin)
error[E0412]: cannot find type `FillVertex` in crate `tessellation`
  --> src/mesh/circle.rs:53:72
   |
53 |             &mut BuffersBuilder::new(&mut mesh, |vertex: tessellation::FillVertex| Vertex {
   |                                                                        ^^^^^^^^^^ not found in `tessellation`

error[E0412]: cannot find type `FillVertex` in crate `tessellation`
  --> src/mesh/rectangle.rs:57:72
   |
57 |             &mut BuffersBuilder::new(&mut mesh, |vertex: tessellation::FillVertex| Vertex {
   |                                                                        ^^^^^^^^^^ not found in `tessellation`

error[E0599]: no method named `hidpi_factor` found for type `winit::window::Window` in the current scope
  --> src/lib.rs:68:31
   |
68 |     let hidpi_factor = window.hidpi_factor();
   |                               ^^^^^^^^^^^^ method not found in `winit::window::Window`

error[E0599]: no method named `to_physical` found for type `winit::dpi::PhysicalSize<u32>` in the current scope
  --> src/lib.rs:69:36
   |
69 |     let size = window.inner_size().to_physical(hidpi_factor);
   |                                    ^^^^^^^^^^^ method not found in `winit::dpi::PhysicalSize<u32>`

error[E0599]: no variant or associated item named `EventsCleared` found for type `winit::event::Event<'_, _>` in the current scope
   --> src/lib.rs:114:27
    |
114 |             event::Event::EventsCleared => {
    |                           ^^^^^^^^^^^^^
    |                           |
    |                           variant or associated item not found in `winit::event::Event<'_, _>`
    |                           help: there is a variant with a similar name: `MainEventsCleared`

error[E0599]: no method named `to_physical` found for type `winit::dpi::PhysicalSize<u32>` in the current scope
  --> src/lib.rs:95:37
   |
95 |                 let physical = size.to_physical(hidpi_factor);
   |                                     ^^^^^^^^^^^ method not found in `winit::dpi::PhysicalSize<u32>`

error[E0609]: no field `width` on type `winit::dpi::Size`
  --> src/system.rs:15:45
   |
15 |         self.screen_descriptor.width = size.width as u32;
   |                                             ^^^^^

error[E0609]: no field `height` on type `winit::dpi::Size`
  --> src/system.rs:16:46
   |
16 |         self.screen_descriptor.height = size.height as u32;
   |                                              ^^^^^^

error[E0599]: no method named `to_physical` found for type `winit::dpi::PhysicalPosition<i32>` in the current scope
   --> src/utils/events.rs:107:41
    |
107 |                 let position = position.to_physical(self.dpi_factor);
    |                                         ^^^^^^^^^^^ method not found in `winit::dpi::PhysicalPosition<i32>`

error: aborting due to 9 previous errors

Some errors have detailed explanations: E0412, E0599, E0609.
For more information about an error, try `rustc --explain E0412`.
error: could not compile `felin`.

To learn more, run the command again with --verbose.
~/personal_projects/rust/vendor/felin (master)$ cargo run --example slider
   Compiling felin v0.1.0 (/home/roman/personal_projects/rust/vendor/felin)
error[E0412]: cannot find type `FillVertex` in crate `tessellation`
  --> src/mesh/circle.rs:53:72
   |
53 |             &mut BuffersBuilder::new(&mut mesh, |vertex: tessellation::FillVertex| Vertex {
   |                                                                        ^^^^^^^^^^ not found in `tessellation`

error[E0412]: cannot find type `FillVertex` in crate `tessellation`
  --> src/mesh/rectangle.rs:57:72
   |
57 |             &mut BuffersBuilder::new(&mut mesh, |vertex: tessellation::FillVertex| Vertex {
   |                                                                        ^^^^^^^^^^ not found in `tessellation`

error[E0599]: no method named `hidpi_factor` found for type `winit::window::Window` in the current scope
  --> src/lib.rs:68:31
   |
68 |     let hidpi_factor = window.hidpi_factor();
   |                               ^^^^^^^^^^^^ method not found in `winit::window::Window`

error[E0599]: no method named `to_physical` found for type `winit::dpi::PhysicalSize<u32>` in the current scope
  --> src/lib.rs:69:36
   |
69 |     let size = window.inner_size().to_physical(hidpi_factor);
   |                                    ^^^^^^^^^^^ method not found in `winit::dpi::PhysicalSize<u32>`

error[E0599]: no variant or associated item named `EventsCleared` found for type `winit::event::Event<'_, _>` in the current scope
   --> src/lib.rs:114:27
    |
114 |             event::Event::EventsCleared => {
    |                           ^^^^^^^^^^^^^
    |                           |
    |                           variant or associated item not found in `winit::event::Event<'_, _>`
    |                           help: there is a variant with a similar name: `MainEventsCleared`

error[E0599]: no method named `to_physical` found for type `winit::dpi::PhysicalSize<u32>` in the current scope
  --> src/lib.rs:95:37
   |
95 |                 let physical = size.to_physical(hidpi_factor);
   |                                     ^^^^^^^^^^^ method not found in `winit::dpi::PhysicalSize<u32>`

error[E0609]: no field `width` on type `winit::dpi::Size`
  --> src/system.rs:15:45
   |
15 |         self.screen_descriptor.width = size.width as u32;
   |                                             ^^^^^

error[E0609]: no field `height` on type `winit::dpi::Size`
  --> src/system.rs:16:46
   |
16 |         self.screen_descriptor.height = size.height as u32;
   |                                              ^^^^^^

error[E0599]: no method named `to_physical` found for type `winit::dpi::PhysicalPosition<i32>` in the current scope
   --> src/utils/events.rs:107:41
    |
107 |                 let position = position.to_physical(self.dpi_factor);
    |                                         ^^^^^^^^^^^ method not found in `winit::dpi::PhysicalPosition<i32>`

error: aborting due to 9 previous errors

Some errors have detailed explanations: E0412, E0599, E0609.
For more information about an error, try `rustc --explain E0412`.
error: could not compile `felin`.

To learn more, run the command again with --verbose.
~/personal_projects/rust/vendor/felin (master)$ rustup show active-toolchain
stable-x86_64-unknown-linux-gnu (default)
~/personal_projects/rust/vendor/felin (master)$ rustc --version
rustc 1.40.0 (73528e339 2019-12-16)
~/personal_projects/rust/vendor/felin (master)$ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=19.10
DISTRIB_CODENAME=eoan
DISTRIB_DESCRIPTION="Ubuntu 19.10"
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -11,7 +11,7 @@ cgmath = "*"
 glsl-to-spirv = "*"
 image = "*"
 winit = "0.20.0-alpha4"
-lyon = "*"
+lyon = "0.13.1"
 collision = "*"
 uuid = { version = "0.7", features = ["serde", "v4"] }
 wgpu = "0.4.0"

